.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+version: 2.1
+aliases:
+  - &defaults
+      docker:
+        - image: cimg/node:12-browsers
+      working_directory: ~/repo
+      environment:
+        NODE_ENV: production
+        RELEASE_TIMESTAMP: "$(date +'%Y%m%d%H%M%S')"
+  - &setup
+    name: "setup"
+    command: |
+      npm --production=false ci
+      mkdir ./test/results
+  - &lint
+    name: "run lint tests"
+    command: npm run lint
+  - &build
+    name: "run npm build"
+    command: |
+      npm run build
+
+jobs:
+  build-test:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          <<: *setup
+      - run:
+          <<: *lint
+      - run:
+          <<: *build
```

Comment:
Seems like this should also run `npm run unit`

Reply:
Yep.  I just updated it.  Good catch.  I think I missed it because Travis calls Test automatically so I didn't noticed that Test calls Unit.